src/HornedBeasts.js

```diff
@@ -1,47 +1,62 @@
 import { Component } from 'react';
 import Image from 'react-bootstrap/Image';
-import Figure from 'react-bootstrap/Figure';
-import FigureImage from 'react-bootstrap/FigureImage';
-import FigureCaption from 'react-bootstrap/FigureCaption';
+import PopUp from './PopUp.js';
+// import Figure from 'react-bootstrap/Figure';
+// import FigureImage from 'react-bootstrap/FigureImage';
+// import FigureCaption from 'react-bootstrap/FigureCaption';
 
 
 class HornedBeasts extends Component {
 
     constructor(props) {
         super(props);
         this.state = {
-            favorites: 0
+            favorites: 0,
+            showModal: false,
+            title: '',
+            image_url: '',
+            key: ''
```

Comment:
Hi there, good work on the lab thus far!

What are your thoughts behind adding title, image_url and key in state? These might work better as props. Grab a TA if ya can. 👍